libs/muddle/src/rbc.cpp

```diff
@@ -34,8 +34,10 @@ constexpr char const *LOGGING_NAME = "RBC";
  * @param threshold Threshold number of Byzantine peers
  * @param dkg
  */
-RBC::RBC(Endpoint &endpoint, MuddleAddress address, CallbackFunction call_back, uint16_t channel)
+RBC::RBC(Endpoint &endpoint, MuddleAddress address, CallbackFunction call_back, uint16_t channel,
+         bool ordered_delivery)
```

Comment:
what is ordered delivery about?

Reply:
The RBC has a counter for all messages sent by a node. In the original RBC protocol messages are delivered in the order of the sequence counter, which means that messages with a greater sequence counter than you expect are held back in a queue until they are can be delivered. I added this option of out of order delivery, where the sequence counter is not checked on delivery for the pre-dkg where you might not have received the original message.